rep-0003.rst

```diff
@@ -233,6 +233,66 @@ Buildsystem support:
 
   - build from source
 
+Lunar Loggerhead (May 2017 - May 2019)
+----------------------------------
```

Comment:
Markdown usually warns you if you don't put an equal number of `-` as the line above it, i.e. you need more here.